src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -177,16 +175,6 @@ public function dump(array $options = array())
         $this->targetDirRegex = null;
         $this->getterProxies = array();
 
-        foreach ($this->classResources as $r) {
-            $this->container->addClassResource($r);
-        }
-        $this->classResources = array();
-
-        foreach ($this->classResources as $r) {
-            $this->container->addClassResource($r);
-        }
-        $this->classResources = array();
```

Comment:
do we actually have this duplicate code in older branches, or was it all added in 3.3 only ?